.drone.yml

```diff
@@ -0,0 +1,611 @@
+---
+kind: pipeline
+name: e2e-kubernetes-1.16
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: init
+    image: quay.io/sighup/e2e-testing-drone-plugin:v0.2.0-rc4
+    pull: always
+    volumes:
+    - name: shared
+      path: /shared
+    depends_on: [ clone ]
+    settings:
+      action: cluster-116
+      pipeline_id: cluster-116
+      instance_path: /shared
+      aws_default_region:
+        from_secret: aws_region
+      aws_access_key_id:
+        from_secret: aws_access_key_id
+      aws_secret_access_key:
+        from_secret: aws_secret_access_key
+      terraform_tf_states_bucket_name:
+        from_secret: terraform_tf_states_bucket_name
+    when:
+      ref:
+        include:
+          - refs/heads/hotfix-*
+          - refs/heads/feature/deprecate-and-update
```

Comment:
The idea is to remove the specific branch when release. I mean, i were developing and i am trying to avoid tag creation to run the pipeline.

The E2E pipeline (once released a stable/tested velero version) should run only:
- On a new tag (both release candidate tagged in a branch or a final release tagged in master)
- While developing a hotfix

Thanks Luca!

Reply:
Perfect, let's remember to update it once the PR is ready to be merged. 😄